systers_portal/meetup/views.py

```diff
@@ -821,3 +821,32 @@ def post(self, request):
             results.sort(key=operator.itemgetter('date'))
             results.sort(key=operator.itemgetter('distance'))
             return JsonResponse({'search_results': results}, safe=False)
+
+
+class AddResourceView(FormValidMessageMixin, FormInvalidMessageMixin, LoginRequiredMixin,
+                      PermissionRequiredMixin, UpdateView):
+    """Add Resources and Images to a past meetup"""
+    template_name = "meetup/edit_meetup.html"
+    model = Meetup
+    slug_url_kwarg = "meetup_slug"
+    form_class = PastMeetup
+    form_valid_message = (u"Meetup edited Successfully")
```

Comment:
Let's change this to something else like "Images have been uploaded"

Reply:
Will do that. Thanks for catching this